contrib/pipelines/secadvisor/main.go

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Missing README that explains how to use. Minimum, what are the command line parameters ?

Reply:
Fixed!